test/test_app_log.c

```diff
@@ -818,7 +823,7 @@ void test_app_log_process_nomem_block (void)
     err_code = app_log_process (&sample);
     TEST_ASSERT (RD_SUCCESS == err_code);
 }
-
+#if     0 // unused functions should not be tested          \/  \/  \/
```

Comment:
Changing unit tests would require a major version bump, and there is no reason to bump major version just because one function is currently unused. Please restore the test

Reply:
I conditionally "off"ed the code to test the 2 (two) functions when the pull request I did to remove them from app_log.c failed.

I would expect that correcting an unit test problem would only require a patch level version change.